recipes/adjfind/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "adjfind" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zwang-geog/AdjFind/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 70411994bb4f1a6f97dcf4e6568b0df08cf20e67aa1e8d6c7fa45a9c80cb02a6
+
+build:
+  number: 0
+  script: |
+    mkdir build
+    cd build
+    cmake ${CMAKE_ARGS} ..  # [unix]
+    cmake %CMAKE_ARGS% ..  # [win]
+    cmake --build . --config Release
+    cmake --install .
```

```suggestion
  script: |
    cmake -B build -S . ${CMAKE_ARGS}  # [unix]
    cmake -B build -S . %CMAKE_ARGS%  # [win]
    cmake --build build --config Release -j ${CPU_COUNT}  # [unix]
    cmake --build build --config Release -j %CPU_COUNT}  # [win]   
    cmake --install build
```
This is a cleaner way to invoke CMake that does not require making or changing directories.